iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/RegulatingPoint.java

```diff
@@ -163,4 +134,27 @@ public void allocateVariantArrayElement(int[] indexes, int sourceIndex) {
             }
         }
     }
+
+    @Override
+    public void onReferencedRemoval(Terminal terminal) {
```

Comment:
could we call the variable removedTerminal? maybe referencedRemoved in the API?

Reply:
Done